android_mobile/src/main/java/com/alexstyl/specialdates/donate/DonateActivity.java

```diff
@@ -81,13 +83,13 @@ public void onClick(View v) {
             @Override
             public void run() {
                 scrollToDonate();
-
             }
 
             private void scrollToDonate() {
                 CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) appBarLayout.getLayoutParams();
                 AppBarLayout.Behavior behavior = (AppBarLayout.Behavior) params.getBehavior();
-                behavior.onNestedFling(coordinator, appBarLayout, null, 0, 50, true);
+                if (behavior != null)
```

Comment:
I would rather understand why the behavior is `null` instead of just checking of null. 

Which version was the emulator? Does that happen with multiple versions of Android?